src/helpers/dimensions.ts

```diff
@@ -166,6 +201,9 @@ export function generateDimensions(
         case VisualizationTypes.LINE: {
             return getLineDimensions(mdObject);
         }
+        case VisualizationTypes.AREA: {
+            return getAreaDimensions(mdObject);
```

Comment:
please check if it's not the same as getBarDimensions, if yes, it could be used as
```
         case VisualizationTypes.BAR:
         case VisualizationTypes.COLUMN:
         case VisualizationTypes.AREA:
             return getBarDimensions(mdObject);
```